hooks/nrpe_helpers.py

```diff
@@ -888,3 +881,69 @@ def set_netlinks_error():
     """Set the flag indicating a netlinks related error."""
     global NETLINKS_ERROR
     NETLINKS_ERROR = True
+
+
+def _get_cis_audit_check():
+    """Get the CIS audit step for nrpe.
+
+    When CIS audit is disabled no step is necessary.
+    """
+    if not hookenv.config("cis_audit_enabled"):
+        return {}
+
+    cis_cmd = cis_cmd_params(include_score=True)
+
+    cis_audit_check = {
+        "description": "Check CIS audit",
+        "cmd_name": "check_cis_audit",
+        "cmd_exec": local_plugin_dir + "check_cis_audit.py",
+        "cmd_params": cis_cmd,
+    }
+    return cis_audit_check
+
+
+def cis_misconfiguration():
```

Comment:
I think the python version here supports type annotations, but I don't see much value adding just for those new functions. We won't be able to run the lint checker because the project is old and all other functions are not signing the types. Moreover, we are adding this new capability because field asked us, but nrpe should be deprecated with COS

Reply:
The value is documenting the return type of the function - if you don't think type annotations are worth it here, then could you consider adding this information to the docstring?  Also the function name could be changed to make the behaviour clearer?

Reply:
Sorry,  I forgot to push the last commit. I've changed the name for your suggestion.
The project, unfortunately,  is not using  `sphinx`. I guess with the change of the name makes it clear

Reply:
Thanks.  It doesn't need sphinx; just informational text in the docstring would be helpful.